internal/safefile/safeopen.go

```diff
@@ -293,12 +293,13 @@ func RemoveAllRelative(path string, root *os.File) error {
 	for {
 		names, err1 := fd.Readdirnames(100)
 		for _, name := range names {
-			err1 := RemoveAllRelative(path+string(os.PathSeparator)+name, root)
+			err1 := RemoveAllRelative(path+string(os.PathSeparator)+name, root) //nolint:govet // shadow
```

Comment:
Should we just rename this `err1` to `err2` to make the intentions clear and also get rid of the warning instead of suppressing it?